nutanix/common_era_schema.go

```diff
@@ -1,12 +1,26 @@
 package nutanix
 
 import (
+	"context"
 	"strconv"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	era "github.com/terraform-providers/terraform-provider-nutanix/client/era"
 )
 
+type dbID string
+
+const dbIDKey dbID = ""
+
+func NewContext(ctx context.Context, dbID dbID) context.Context {
```

Comment:
Can we also add a comment in the code explaining why is this done?

Reply:
sure.